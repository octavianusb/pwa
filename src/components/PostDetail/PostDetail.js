import React from 'react'

// COMPONENTS
import Post from '../Post/Post'

export default function PostDetail (props) {
  return (
    <div><Post {...props.post[0]} /></div>
  )
}
