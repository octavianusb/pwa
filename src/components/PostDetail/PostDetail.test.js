import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import PostDetail from './PostDetail'

describe('PostDetail component', () => {
  it('should render as expected', () => {
    const component = shallow(<PostDetail />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})