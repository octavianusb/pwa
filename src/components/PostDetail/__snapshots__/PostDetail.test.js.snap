// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostDetail component should render as expected 1`] = `
<div>
  <Post />
</div>
`;
